passes/pmgen/peepopt_muldiv_c.pmg

```diff
@@ -48,12 +48,26 @@ code
 
 	// Get properties and values of b_const and c_const
 	int b_const_width = mul->getParam(ID::B_WIDTH).as_int();
+	// b_const may be coming from the A port but it's an RTLIL invariant that A_SIGNED equals B_SIGNED
 	bool b_const_signed = mul->getParam(ID::B_SIGNED).as_bool();
 	bool c_const_signed = div->getParam(ID::B_SIGNED).as_bool();
 	int b_const_int = b_const.as_int(b_const_signed);
 	int c_const_int = c_const.as_int(c_const_signed);
 	int b_const_int_shifted = b_const_int << offset;
 
+	// Integer values should be lesser than 64 bits
+	if (mul->getParam(ID::B_WIDTH).size() > 64)
+		reject;
+	if (b_const.size() > 64)
+		reject;
```

Comment:
I keep forgetting this but I think the typical size of int (and the standard mandated minimum) is 32 bits, also let's check `b_const.size() + offset`

Reply:
Done here: https://github.com/akashlevy/yosys/pull/13